### PR TITLE
[DEV-1604] correct onlyMinter modifier

### PR DESCRIPTION
**Pull request by waynehoover**

### CODE REVIEW

**Review by jonathandiep — APPROVED** [review]

**Review by GarrettJMU — APPROVED** [review]

**Comment by GarrettJMU** [comment]
Looks like you have a failing test 👀

File: test/Erc20Quest.spec.ts [hunk 1]
```diff
@@ -335,32 +335,36 @@ describe('Erc20Quest', async () => {
...
-      await deployedQuestContract.connect(firstAddress).claim()
-
+      await deployedQuestContract.connect(owner).claim()
       await ethers.provider.send('evm_increaseTime', [expiryDate + 1000])
       await deployedQuestContract.withdrawRemainingTokens(owner.address)
 
       expect(await deployedSampleErc20Contract.balanceOf(deployedQuestContract.address)).to.equal(200)
       expect(await deployedSampleErc20Contract.balanceOf(owner.address)).to.equal(
-        totalRewardsPlusFee * 100 - 1 * 1000 - 200
+        // totalRewardsPlusFee * 100 - 1 * 1000 - 200
```
Comment (waynehoover) [inline test/Erc20Quest.spec.ts:349]: I had to change the math here. @GarrettJMU or @jonathandiep is this correct? Not sure if it was incorrect before or now.

**Review by GarrettJMU — APPROVED** [review]

File: test/Erc20Quest.spec.ts [hunk 2]
```diff
@@ -334,33 +334,39 @@ describe('Erc20Quest', async () => {
...
-      await deployedFactoryContract.connect(firstAddress).mintReceipt(questId, messageHash, signature)
+    it('should transfer correct amount non-claimable rewards to address', async () => {
+      await deployedRabbitholeReceiptContract.setMinterAddress(deployedFactoryContract.address)
+
+      await deployedFactoryContract.mintReceipt(questId, messageHash, signature)
       await deployedQuestContract.start()
       await ethers.provider.send('evm_increaseTime', [expiryDate + 100])
-      await deployedQuestContract.connect(firstAddress).claim()
-
+      await deployedQuestContract.connect(owner).claim()
```
Comment (jonathandiep) [inline test/Erc20Quest.spec.ts:343]: We should try to move the `claim()` to after `withdrawRemainingTokens()` cause we want to ensure that even after withdraws that users can still claim rewards if they haven't

Comment (waynehoover) [reply test/Erc20Quest.spec.ts]: Good idea, I modified the test to do that.

**Pull request merged**